content/articles/picking-pdf-and-image-from-phone-storage/index.md

```diff
@@ -0,0 +1,322 @@
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="48dp"
+        android:text="Click To Select Image from Storage"
+        android:textColor="#03A9F4"
+        android:textSize="22sp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <TextView
+        android:id="@+id/selectedPdf"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="248dp"
+        android:layout_marginEnd="16dp"
+        android:padding="20dp"
+        android:text="Click To Pick PDF From Storage"
+        android:textColor="#0798DA"
+        android:textSize="22sp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/imageView" />
+```
+### Picking Image
+To open the phone's gallery you need an intent to handle the action. You will click on the `TextView` that will pop up an alert dialog having options to take a photo or choose an image from the gallery. You will also learn how to trigger alert dialogs in this tutorial. 
+
+Add the following function to your code to display the alert dialog for selecting an image from the gallery or taking a photo.
+
+```kotlin
+    private fun selectImage() {
+        val choice = arrayOf<CharSequence>("Take Photo", "Choose from Gallery", "Cancel")
+        val myAlertDialog: AlertDialog.Builder = AlertDialog.Builder(this)
+        myAlertDialog.setTitle("Select Image")
+        myAlertDialog.setItems(choice, DialogInterface.OnClickListener { dialog, item ->
+            when {
+                choice[item] == "Choose from Gallery" -> {
+                    val pickFromGallery = Intent(Intent.ACTION_GET_CONTENT, MediaStore.Images.Media.EXTERNAL_CONTENT_URI)
+                    pickFromGallery.type = "/image"
+                    startActivityForResult(pickFromGallery, 1)
+                }
+                choice[item] == "Take Photo" -> {
+                    val cameraPicture = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
+                    startActivityForResult(cameraPicture, 0)
+                }
+                choice[item] == "Cancel" -> {
+                    myAlertDialog.dismiss()
+                }
+            }
+        })
+        myAlertDialog.show()
+    }
+```
+Below is a screenshot of the alert dialog.
+
+[Alert Dialog](/engineering-education/picking-pdf-and-image-from-phone-storage/alert-dialog.png)
+
+When you want only the images in your gallery to be displayed, you include the type of intent to be images but if you want to display videos as well do not specify the type. 
```

Comment:
Change to " When you want only the images in your gallery to be displayed, you include the intent to be images, but if you want to display videos, do not specify the type."